code/script.js

```diff
@@ -1,9 +1,17 @@
+console.log('HELLO')
+
 // A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
 function toggle() {
   this.classList.toggle("active")
 }
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
+// Selects a "section", and calls a function which will be executed when the element is clicked.
 document.getElementById("section1").onclick = toggle
+document.getElementById("section2").onclick = toggle
+document.getElementById("section3").onclick = toggle
+document.getElementById("section4").onclick = toggle
+
+// Prevent the information from disappearing.
+document.getElementById('form').onsubmit = event => {
+  //event.preventDefault()
```

Comment:
I'm not sure if you wanted to keep as the code inside the function is commented out? Nice touch to the form by the way! :) I should definitely learn more about this or rewatch the recorded live one more time to get a better understanding. Thank you!

Reply:
Oh it wouldn't be "out-commented". But no, it wasn't necessary for the function. More that it doesn't reload again but "keep" the information from the function..